recipes-example/template-service/template-service.bb

```diff
@@ -0,0 +1,23 @@
+#
+#   Copyright (C) 2017 Pelagicore AB
+#
+
+DESCRIPTION = "A template C/C++ service"
+HOMEPAGE = "https://github.com/GordanM/template-service.git"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+
+PR = "r0"
+PV = "1.0+git${SRCREV}"
+
+SRC_URI = "gitsm://github.com/GordanM/template-service.git;protocol=https;branch=topic/template-library"
```

Comment:
I don't think this uses git submodules either, you should replace it with git://